doc/07-Operations.md

```diff
@@ -0,0 +1,128 @@
+It covers several potential errors, including operations that take too long or invalid high availability scenarios.
+Even if the Icinga DB has crashed, checks will still run and Icinga 2 would generate notifications.
+
+In addition, both the Redis® and the relational database should be monitored.
+There are predefined check commands in the ITL to choose from.
+
+- [`redis`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#redis)
+- [`mysql`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#mysql)
+- [`mysql_health`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#mysql_health)
+- [`postgres`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#postgres)
+
+A simpler approach would be to check if the processes are running, e.g.,
+with [`proc`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#procs) or [`systemd`](https://icinga.com/docs/icinga-2/latest/doc/10-icinga-template-library/#systemd).
+
+## Backups
+
+There are only two things to back up in Icinga DB.
+
+1. The configuration file in `/etc/icingadb` and
+2. the relational database, using `mysqldump`, `mariadb-dump` or `pg_dump`.
+
+!!! warning
+
+    When creating a database dump for MySQL or MariaDB with `mysqldump` or `mariadb-dump`,
+    use the [`--single-transaction` command line argument flag](https://dev.mysql.com/doc/refman/8.4/en/mysqldump.html#option_mysqldump_single-transaction)
+    to not lock the whole database while the backup is running.
+
+## Automatic Restart
+
+<!-- NOTE: Would be obsolete after https://git.icinga.com/packages/icingadb/-/merge_requests/10 -->
+
+The retry logic of the Icinga DB daemon will retry certain errors for up to five minutes before giving up and stopping the daemon.
+While an error persisting for five minutes in most cases indicates a critical error,
+in some environments this happens occasionally and continuing is a valid solution.
+
+In this case, the `icingadb.service` unit can be modified to include a [`Restart` option](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Restart=).
+
+!!! warning
+
+    Making systemd retry fatal Icinga DB errors may help in some cases, but it usually hides the symptom of a deeper problem.
+    Please do not use this as a silver bullet and try to understand the error, search for already [reported issues](https://github.com/Icinga/icingadb/issues) or report one yourself.
+
+```shell
+systemctl edit icingadb.service
+```
+
+Add a `[Service]` block like the following.
+
+```ini
+[Service]
+Restart=on-failure
+RestartSec=60
+```
+
+This will create an additional `/etc/systemd/system/icingadb.service.d/override.conf` file that overrides the defaults.
+A final unit restart is required for the changes to take effect.
+
+```shell
+systemctl restart icingadb.service
+```
```

Comment:
I'd separate this section from the PR, otherwise I see the risk of this part stalling the entire PR. We discussed the retry issue yesterday and wanted to analyze the circumstances that lead to situations where stopping after 5 minutes wasn't a good idea. Until that's done, I wouldn't add a section recommending `Restart=on-failure` here.

Reply:
Removed and stashed to be created as a draft PR later on.

Reply:
To be continued in #958.